src/child/containers/App.js

```diff
@@ -18,8 +17,14 @@ require('script!../../../node_modules/BitFlux/node_modules/bootstrap/js/dropdown
 require('script!../../../node_modules/d3fc/dist/d3fc.bundle.min.js');
 require('script!../../../node_modules/BitFlux/dist/bitflux.js');
 
+const contextMenu = (e) => {
+    if (process.env.NODE_ENV === 'production') {
+        e.preventDefault();
+    }
+};
+
 const App = ({ code, name }) => (
-    <div className="main">
+    <div className="main" onContextMenu={contextMenu}>
```

Comment:
I've had a closer look - I think the OpenFin config can be used for this.

The [ConfigService determines if the context menu should be shown](https://github.com/ScottLogic/StockFlux/blob/72e040fafec7b38d3dda1bd93824107118f4517b/src/shared/ConfigService.js#L14); this was previous updated by a Grunt task, but we can just update to use `process.env.NODE_ENV`.